src/ripple/app/tx/impl/Transactor.h

```diff
@@ -170,6 +170,11 @@ class Transactor
 
     virtual TER doApply () = 0;
 
+    static
```

Comment:
A javadoc comment explaining exactly what this function does would be great.

Reply:
Fixed. Let me know if that works.